docs/products/mysql/concepts/mysql-memory-usage.rst

```diff
@@ -0,0 +1,16 @@
+MySQL memory usage
+==================
+
+MySQL InnoDB buffer pools are **pre-allocated** and shown as **used memory** even without user activity or stored data in the database. Because of this pre-allocated memory, it's expected high memory usage within all Aiven MySQL plans. 
```

Comment:
```suggestion
`MySQL InnoDB buffer pools <https://dev.mysql.com/doc/refman/8.0/en/innodb-buffer-pool.html>`_ are **pre-allocated** and shown as **used memory** even without user activity or stored data in the database. Because of this pre-allocated memory, you can expect a high memory usage within all Aiven MySQL plans. 
```

Reply:
As it is the same link and they are very close maybe I will add only once